nighthawk/source/client/utility.cc

```diff
@@ -0,0 +1,64 @@
+
+#include "absl/strings/match.h"
+#include "absl/strings/str_split.h"
+
+#include "common/http/utility.h"
+#include "common/network/utility.h"
+
+#include "nighthawk/source/common/utility.h"
+
+namespace Nighthawk {
```

Comment:
Why is the `.cc` in `client` but the `.h` is in `common`?

Reply:
Whoops, moved the `.cc` to `common` so it lives next to the `.h`.